test/stubs.ts

```diff
@@ -244,16 +248,40 @@ export class ProjectDataStub implements IProjectData {
 	get platformsDir(): string {
 		return "";
 	}
+	set platformsDir(value) {
+	}
 	projectFilePath: string;
 	projectId: string;
 	dependencies: any;
 	appDirectoryPath: string;
-	appResourcesDirectoryPath: string;
 	devDependencies: IStringDictionary;
 	projectType: string;
-	initializeProjectData(projectDir?: string): void {
+	get appResourcesDirectoryPath(): string {
+		return this.getAppResourcesDirectoryPath();
+	}
+	public initializeProjectData(projectDir?: string): void {
 		this.projectDir = this.projectDir || projectDir;
 	}
+	public getAppResourcesDirectoryPath(projectDir?: string): string {
```

Comment:
I found it easier to test this way, since I would always get the default path to the App_Resources. Ideally I could always resolve the path to the default app/App_Resources for the purposes of testing.